Makefile

```diff
@@ -70,6 +71,47 @@ deploy: manifests
 # Generate manifests e.g. CRD, RBAC etc.
 manifests:
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
+	# Config CRDs
+	awk 'FNR==1 && NR!=1 {print "---"}{print}' config/crds/*.yaml > $(CHARTDIR)/templates/_crds.yaml
+	yq m -d'*' -i $(CHARTDIR)/templates/_crds.yaml hack/chart-metadata.yaml
```

Comment:
Could we change these customizations to using Kustomize?

yq appears to be an ambiguous command (two versions from two different authors) and isn't in any distro's repository.

Kustomize was made for this sort of thing. It wouldn't be able to perform the deletions.

Reply:
Sylus, Are you planning to handle this? Checked build logs and they are failing with `yq` not found error.

I'm happy to resolve this. Let me know. 

cc @ritazh  @maxsmythe

Reply:
Hi @jayapriya90 and @sylus, I am currently working on switching this to use kustomize to remove the yq dependency.

Reply:
Thanks @ctab !